src/main/kotlin/org/rust/ide/core/overrideImplement/ImplementMembersHandler.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.ide.core.overrideImplement
+
+import com.intellij.lang.LanguageCodeInsightActionHandler
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsImplItem
+import org.rust.lang.core.psi.impl.RsFile
+import org.rust.lang.core.psi.util.parentOfType
+
+class ImplementMembersHandler: LanguageCodeInsightActionHandler {
+    override fun isValidFor(editor: Editor, file: PsiFile): Boolean {
+        if (file !is RsFile)
+            return false
```

Comment:
Let's either format this as a single line `if (file !is RsFile) return` or add braces to avoid goto fail bug?